rudder-web/src/main/webapp/secure/administration/apiManagement.html

```diff
@@ -118,7 +118,15 @@ <h1 id="title" style="color: #000; wrap-word:break-word;">{{formTitle(myNewAccou
                       <div class="simplemodal-bottom">
                         <div class="popupButton" style="width: 98%;border-top: 1px solid #9A9A9A;margin-top: 10px;">
                           <span>
-                            <input type="submit" class="btn-primary" style="float:right;" value="Save" ng-disabled="newAccount.$pristine || newAccount.$invalid" ng-click="checkAndSaveAccount(myNewAccount,myNewAccount.index,newAccount);newAccount.$setPristine();" />
+                            <button
```

Comment:
The angular app was started twice in a row :
1. with the ng-app directive (<div class="portlet" ng-app="accountManagement" [...]>)
2. with : angular.bootstrap(document.getElementById('accountManagement'), ['accountManagement']);

This caused a JS error which was the origin of this bug